src/Microsoft.DocAsCode.Build.Engine/MarkdownProviders/DfmServiceProvider.cs

```diff
@@ -157,6 +157,11 @@ public sealed class DfmService : IMarkdownService, IHasIncrementalContext, IDisp
             }
 
             public MarkupResult Markup(string src, string path)
+            {
+                return Markup(src, path, false);
+            }
+
+            public MarkupResult Markup(string src, string path, bool enableValidation)
```

Comment:
>enableValidation [](start = 69, length = 16)

It's not implemented here.

Reply:
Add some comments if it's costly to implement now.

---
In reply to: [186004316](https://github.com/dotnet/docfx/pull/2726#discussion_r186004316) [](ancestors = 186004316)